packages/connection/src/Manager.php

```diff
@@ -21,6 +21,11 @@ class Manager implements Manager_Interface {
 	const SECRETS_OPTION_NAME    = 'jetpack_secrets';
 	const MAGIC_NORMAL_TOKEN_KEY = ';normal;';
 	const JETPACK_MASTER_USER    = true;
+	// These constants can be set in wp-config.php to ensure sites behind proxies will still work.
```

Comment:
I don't think we need these at all anymore, do we?

Can we move the comments to jetpack.php and get rid of these class constants entirely?

Reply:
how can we be sure that a plugin depending on this has set them?
Sure we are sure here in the newly minted mono-repo of jetpack
but outside of jetpack, can we know for certain?